### PR TITLE
agent: document AWS X-Ray exporter

### PR DESCRIPTION
**Pull request by odeke-em**
Updates #560

### CODE REVIEW

File: content/agent/exporters/aws_xray/_index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    default_service_name: "ocagent"
+    version: "latest"
+    buffer_size: 200
+```
+
+### End to end example
+
+In this end-to-end example, we'll have ocagent running and a couple of Go applications
+that use the [Go ocagent-exporter](/exporters/supported-exporters/go/ocagent)
+to send over traces to ocagent and then to aws_xray.
```
Comment (mayurkale22) [inline content/agent/exporters/aws_xray/_index.md]: ```s/aws_xray/AWS X-Ray```?

File: content/agent/exporters/aws_xray/_index.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+#0: LineLength: 292By
+#1: LineLength: 339By
+Latency: 669.318ms
+#0: LineLength: 412By
+Latency: 86.700ms
+```
+
+### Results
+
+On navigating to the AWS X-Ray at console.aws.amazon.com/xray/home
```
Comment (mayurkale22) [inline content/agent/exporters/aws_xray/_index.md]: console.aws.amazon.com/xray/home -> ```[https://console.aws.amazon.com/xray/home](https://console.aws.amazon.com/xray/home)```

File: content/agent/exporters/aws_xray/_index.md (hunk 1 above)
Comment (odeke-em) [reply content/agent/exporters/aws_xray/_index.md]: Roger that, thanks!

**Comment by odeke-em** [comment]
Cool, thank you @songy23 and @mayurkale22 for the reviews!

**Pull request merged**